README.md

```diff
@@ -1,166 +1,88 @@
-<h1>
-  <a href="https://www.sanity.io">
-    <img width="40%" alt="Sanity Logo" src="https://www.sanity.io/static/images/logo_red.svg?v=2"/>
-  </a>
-</h1>
+# ![Sanity logo](https://cdn.sanity.io/images/3do82whm/next/51af00784c5addcf63ae7f0c416756acca7e63ac-353x71.svg?w=180&fm=png&fit=max) – Build with Structured Content
 
-<h4>
-  <a href="https://www.sanity.io">Sanity</a> is a real-time content infrastructure. The editor, built in JavaScript and React.js, lives in this repo. It connects to a scalable, hosted backend featuring a Graph Oriented Query Language (GROQ), asset pipelines and fast edge caches.
-</h4>
+> **Sanity.io is the platform for structured content, that comes with an open-souce editing environment called Sanity Studio that you can customize with JavaScript, and a hosted real-time data store.**
 
-<p>
-  <a href="https://slack.sanity.io/"><img src="https://slack.sanity.io/badge.svg"></a>
-</p>
+![npm](https://img.shields.io/npm/v/@sanity/base) ![Downloads](https://img.shields.io/npm/dm/@sanity/client) [![](https://slack.sanity.io/badge.svg)](https://slack.sanity.io/) ![Follow on twitter](https://img.shields.io/twitter/follow/sanity_io?style=flat&color=blue)
 
-## Table of contents
-
-- [Table of contents](#table-of-contents)
-- [Getting started](#getting-started)
-- [Key Features](#key-features)
-- [Plugins and tooling](#plugins-and-tooling)
```

Comment:
Don't we want to list plugins directly from the  Github?

Reply:
I don't think we should try having all of them, so a selection perhaps?

Reply:
I was thinking about just linking to plugins? Again – sry about lack of specificity.

Reply:
Ah, directly from the ToC?